lib/sisimai/smtp/error.rb

```diff
@@ -85,17 +79,17 @@ def soft_or_hard(argv1 = '', argv2 = '')
             softorhard = ''
 
           elsif argv1 == 'onhold' || argv1 == 'undefined'
-            # Check with the value of D.S.N. in $argv2
+            # Check with the value of D.S.N. in argv2
             getchecked = Sisimai::SMTP::Error.is_permanent(argv2)
 
             if getchecked.nil?
               softorhard = ''
             else
               softorhard = if getchecked
-                'hard'
-              else
-                'soft'
-              end
+                             'hard'
+                           else
+                             'soft'
+                           end
```

Comment:
__[rubocop]__
* Lint/EndAlignment: `end` at 92, 40 is not aligned with `if` at 88, 27.

<!-- Comment by [SideCI](https://sideci.com) -->